packages/vscode-extension/src/utilities/extensionContext.ts

```diff
@@ -1,11 +1,13 @@
 import fs from "fs";
-import { commands, ExtensionContext, Uri, workspace, window, Disposable } from "vscode";
+import { commands, ExtensionContext, Uri, workspace, window } from "vscode";
 import { Logger } from "../Logger";
 import { getLaunchConfiguration } from "./launchConfiguration";
 import { LaunchConfigurationOptions } from "../common/LaunchConfig";
 
 let _extensionContext: ExtensionContext | null = null;
 
+type searchItem = { path: string; searchDepth: number };
```

Comment:
NIT:
```suggestion
type SearchItem = { path: string; searchDepth: number };
```
Capitalizing types makes the code easier to read at a glance.

Reply:
good catch